### PR TITLE
Test for custom object-returning inspect

### PR DESCRIPTION
**Pull request by flaambe**

### CODE REVIEW

**Comment by keithamus** [comment]
Oh interesting, we need to recurse if not a string I guess?

**Comment by keithamus** [comment]
Added a fix. Let me know what you think and can merge.

File: test/acceptance.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ describe('objects', () => {
     const obj = {
       inspect: () => 1,
     }
-    expect(inspect(obj, { customInspect: true })).to.equal(1)
+    expect(inspect(obj, { customInspect: true })).to.equal('1')
```
Comment (keithamus) [inline test/acceptance.js:33]: This is probably a good change because we ideally want loupe to always return a string.

**Comment by flaambe** [comment]
Seems ok, but the coverage has decreased

**Comment by keithamus** [comment]
I think the coverage reporting is broken because a browser has failed to execute.

**Review by keithamus — APPROVED** [review]

**Pull request merged**